lib/xpm_ruby/connection.rb

```diff
@@ -3,11 +3,12 @@
 
 module XpmRuby
   class Connection
-    attr_accessor :xero_tenant_id, :authorization
+    attr_accessor :xero_tenant_id, :authorization, :url
 
-    def initialize(access_token:, xero_tenant_id:)
+    def initialize(access_token:, xero_tenant_id:, url: nil)
       @xero_tenant_id = xero_tenant_id
       @authorization = "Bearer " + access_token
+      @url = url || xpm_url
```

Comment:
Alternative style part 2:
```suggestion
    @url = url
```